i18n/bg/camera.ftl

```diff
@@ -0,0 +1,78 @@
+app-title = Камера
+about = Относно
+repository = Хранилище
+view = Изглед
+welcome = Добре дошли в COSMIC! ✨
+page-id = Страница { $num }
+git-description = Git подаване {$hash} на {$date}
+
+# Mode switcher
+mode-video = ВИДЕО
+mode-photo = СНИМКИ
+mode-virtual = ВИРТУАЛНО
+
+# Virtual camera
+virtual-camera-title = Виртуална камера (експериментално)
+virtual-camera-description = Предавайте потока от камерата си към други програми чрез виртуално устройство за камера. Изисква PipeWire.
+virtual-camera-enable = Активиране на виртуалната камера
+streaming-live = НА ЖИВО
+virtual-camera-open-file = Отваряне на файл
+virtual-camera-file-filter-name = Изображения и видеа
+
+# Filters
+filters-title = Филтри
+
+# Settings
+settings-title = Настройки
+settings-camera = Камера
+settings-video = Видео
+settings-device = Устройство
+settings-format = Формат
+settings-microphone = Микрофон
+settings-encoder = Кодер
+settings-quality = Качество
+settings-video-encoder = Видео кодер
+settings-video-quality = Качество на видео
+settings-manual-override = Manual mode override
```

Comment:
Thanks for adding Bulgarian translations! I noticed settings-manual-override on line 36 is still in English. Was this intentional, or would you like to add a Bulgarian translation for it?